content/soarian/dstu2/diagnostic/diagnostic-report.md

```diff
@@ -0,0 +1,94 @@
+
+The API returns the test names and values in reverse chronological order by result date and, within that, the sequence that is configured in the Soarian flowsheet that defines the qualified tests.  Qualified results of status Final, Corrected, or Cannot Obtain are returned.  Result values of numeric or text are returned; image only results, including those encapsulated in non-text formats such as pdf, are not returned.  Since each result is individually dated and the value is unambiguously relevant for that date, this resource supports date-based queries.
+
+## Terminology Bindings      
+
+
+<%= terminology_table(:soarian_diagnostic_report, :dstu2) %>
+
+## Extensions
+The content of these extensions must be visible to the user, and recorded with any storage or transmit activity along with the query/response date/time, as it represents the best representation of what clinicians recorded for the patient at the time of query.  
+
+
+| Name | Condition | Description |
+| --- | --- | --- |
+|[Narrative](https://www.hl7.org/fhir/narrative.html)|No qualifying entries exist|**No relevant clinical data exists**|
+|[Narrative](https://www.hl7.org/fhir/narrative.html)|Qualifying entries exist|The following labels and values from the list will be returned in HTML, if valued: **Test/Procedure Name, Test Value, Value Unit, Reference Range, micro (culture) results**|
+|[Narrative](https://www.hl7.org/fhir/narrative.html)|Qualifying entries exist|Informational message that must be shown to the user: **Information provided is from YYYY-MM-DD through YYYY-MM-DD inclusive.**|
+
+**CAUTION:** The detail that is in the narrative may not be fully expressed in the structured portion of the FHIR response: The https://www.hl7.org/fhir/narrative-definitions.html#Narrative.status entry in the narrative is valued as "Additional" indicating that more information is provided in the narrative than is provided in the structure. Therefore, if the Application is using the structured entries for any purpose, care must be taken to ensure the clinical content/context of the narrative is preserved.  
+
+## Search
+
+### Parameters
+
+No DiagnosticReport resource specific search parameters are supported. Date / Date range is an optional search parameter.
+
+|Name |Required? | Type | Description
+| --- | --- | --- | --- |
+| patientId | Y | reference | The patient identifier provided in a pre-requisite authorization step. |
+| date | N | [date](http://hl7.org/fhir/DSTU2/search.html#date) as adjusted per implementation notes | Null or specific date or a date range. |
+
+Implementation notes.
+
+While the FHIR DSTU date type is defined, the API relies on the HAPI-FHIR development framework which has certain date behaviors that the developer should be aware of during implementation.  Since these differ in some ways from DSTU2, they are elaborated in detail [here] [date-Param]
```

Comment:
No need to have this here - just link above in the description to the section in the overview.